### DISCUSSION TITLE
“Are you sure you want to leave this page?” confirmation on exit

### DISCUSSION THREAD

**Discussion by stuymedova** [community/community#20468] [Projects and Issues]
In cases where user exits an issue/discussion webpage (by closing browser tab) after having entered issue/discussion content, I suggest to ask for confirmation to prevent accidental deletion of said content.

**Comment by radar-blip** [comment]
Don't you get confirmation of some kind already? Maybe I'm misunderstanding.

Anyway, if you wish you can file a bug report here: 
https://support.github.com/contact/bug-report
